src/components/BreedsList.vue

```diff
@@ -256,39 +255,40 @@ export default {
   },
   watch: {
     $route(newRoute, oldRoute) {
+      console.log("watch route");
       if (this.$route.name === "breedName") {
         this.$router.push({
           name: "details",
         });
       }
+      console.log("new", newRoute, "old", oldRoute);
+      console.log(newRoute);
+
+      this.lastBreedState = [];
+      this.indexClicked = 0;
+      this.displayedBreeds.forEach((breed) => {
+        breed.isActive = false;
+      });
+    },
+
+    displayedBreeds: {
+      handler(breeds) {
+        breeds.filter((breed, id) => {
+          if (breed.isActive) {
+            this.$refs.resultListContainer.children[id].scrollIntoView({
+              behavior: "smooth",
+              block: "center",
+            });
+          }
+        });
+      },
+      deep: true,
```

Comment:
Sedaj sem namesto `updated `hooka uporabil deep `watcher `na `displayedBreeds `computed ki vrze ali `searchList `arr ali pa `breedsList `arr. in znotraj tega sem potem uporabil `scrollIntoView` fn, ki naj bi scrollalo na sredino `ul`. Ampak pa ne scrolla na sredino, vedno scrolla od spodnjega oz zgornjega roba li? Mogoče ker li nima fiksne višine? Ampak pri updated hooku je delalo.